source/one-or-more.d.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/array-type */
+/**
+Either matches an array containing only `Value`, or `Value` itself, not in an array.
+
+@example
+import {OneOrMore} from 'type-fest';
+
+type ID = number;
+type OneOrMoreIDs = OneOrMore<ID>;
+
+const multipleIDs: OneOrMoreIDs = [1234, 2468];
+const oneID = OneOrMoreIDs = 1234;
```

Comment:
```suggestion
const oneID: OneOrMoreIDs = 1234;
```

Reply:
FFS I'm sure I did this, must've undid it somehow.